KronorComponents/Common/EmbeddedPaymentViewModel.swift

```diff
@@ -161,6 +165,10 @@ class EmbeddedPaymentViewModel: ObservableObject {
                         merchantReturnURL: self.returnURL,
                         device: self.device
                     )
+                case .bankTransfer:
+                    // cannot create the payment request as we don't know how.
+                    // it will be created by the web version of the payment gateway
+                    fatalError("impossible")
                 case .fallback(_):
```

Comment:
We could add the new `.bankTransfer` case to the other one to avoid duplication. Additionally I notice that for the current fallback case we can omit the parenthesis. So I would suggest changing this case to: 
`case .fallback, .bankTransfer:`